app/_locales/de/messages.json

```diff
@@ -2061,7 +2061,7 @@
   },
   "permission_ethereumAccounts": {
     "message": "Siehe Adresse, Kontostand, Aktivität und Einleitung von Transaktionen",
-    "description": "The description for the `eth_accounts` permission"
+    "description": "The description for the `permission_ethereumAccounts` permission"
```

Comment:
[nit] I don't think these messages need descriptions. We typically use descriptions to give some context around placeholders (`$1`, `$2`, etc.) that we place in messages, but these messages don't have placeholders, so I think the description is redundant and you could remove them if you wanted to.

Reply:
The description field should be encouraged for all messages, because they provide more context for translators on how the message is used. There is an open issue for requiring them for all new messages: https://github.com/MetaMask/metamask-extension/issues/10461

Reply:
Though in this case, this description is not being added in this PR. It looks like it wasn't even meant to be changed, and that this was done by accident. This is what's causing the lint failure on CI.

The permission is still called `eth_accounts`, so the old version is correct.

Reply:
> The description field should be encouraged for all messages

Ah okay, good to know. Hadn't seen that issue.